components/FolderButton.tsx

```diff
@@ -0,0 +1,30 @@
+import { styled } from "styled-components";
+
+type props = {
+  children: React.ReactNode;
+  request: () => void;
```

Comment:
이렇게 표현해주는게 더 좋을 것 같네요!
```suggestion
  onClick: () => void;
```